README.md

```diff
@@ -1 +1,19 @@
-# Fill me in with information
+Official MongoDB Node.js Driver Quick Start
+===========================================
+
+You can use this starter project to follow along the
+[MongoDB Node.js Driver Quick Start](https://docs.mongodb.com/drivers/node/current/quick-start/).
+
+This project contains a `package.json` file which contains information
+about the project and its dependencies. The npm package manager uses the
+information in this file to install any dependencies and run your Node.js
+application. Learn more about the npm package manager on the [npm Docs
+site](https://docs.npmjs.com/about-npm).
+
+
+Getting Started
+===============
+
+Follow [Set up Your Project](https://docs.mongodb.com/drivers/node/current/quick-start/#set-up-your-project)
```

Comment:
Comment: I think this may be missing a "the"?

```suggestion
Follow the [Set up Your Project](https://docs.mongodb.com/drivers/node/current/quick-start/#set-up-your-project)
```